translations/fr/try_ruby_120.md

```diff
@@ -0,0 +1,21 @@
+---
+lang:   EN
+title:  Faire la queue
+answer: ^\[(\d+)(,\s*\d+){3,}\]$
+ok:     Excellent
+error:  Essaye d'ajouter un quatrième nombre à la liste
+---
+
+Les listes (arrays) stockent des choses __dans un ordre indéfini__.
```

Comment:
Here the point is actually about having `un ordre défini` rather than `un ordre indéfini`, isn't it? Arrays _do_ have an order.

Reply:
Yep.